llvm/test/CodeGen/Mips/ins.ll

```diff
@@ -0,0 +1,39 @@
+; RUN: llc -O3 -mcpu=mips64r2 -mtriple=mips64el-unknown-linux-gnuabi64 < %s -o - | FileCheck %s
```

Comment:
Since you are testing `INS ` , mips64 is not required. Please use a mips32 triple.

Reply:
OK, I would update the test.